tests/foreman/ui/test_hostgroup.py

```diff
@@ -23,10 +23,15 @@
 from robottelo.ui.factory import make_hostgroup
 from robottelo.ui.locators import common_locators
 from robottelo.ui.session import Session
+from robottelo.config import settings
 
 
 class HostgroupTestCase(UITestCase):
     """Implements HostGroup tests from UI"""
+    @classmethod
+    def setUpClass(cls):
+        super(HostgroupTestCase, cls).setUpClass()
```

Comment:
Docstrings are required for all the methods. Please add simple 1-line docstring describing the purpose of defining `setUpClass` here :)